<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForcesWidget</class>
 <widget class="QWidget" name="ForcesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Local run</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="Mesh_and_Solver">
     <property name="title">
      <string>Linkers</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Simulation</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Mesh</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comb_Mesh"/>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comb_Simulation"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Solver-Material Mapping:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tw_Solver"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
